telemetry/definitions/commonDefinitions.json

```diff
@@ -30,6 +30,11 @@
             "type": "string",
             "description": "The reason for a metric or exception depending on context"
         },
+        {
+            "name": "eventbridgeschema",
+            "type": "string",
+            "description": "The name of the EventBridge Schema used in the operation"
```

Comment:
What is gained by having these hyper-specific types? Why not just use string?

Reply:
we could but the type also sets the name, it doesn't actually generate an `enum` or anything else for it in Kotlin. In typescript it generates comments of what each type is.